Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/restclient/request.go

```diff
@@ -335,13 +335,13 @@ type versionToResourceToFieldMapping map[unversioned.GroupVersion]resourceTypeTo
 func (v versionToResourceToFieldMapping) filterField(groupVersion *unversioned.GroupVersion, resourceType, field, value string) (newField, newValue string, err error) {
 	rMapping, ok := v[*groupVersion]
 	if !ok {
-		glog.Warningf("Field selector: %v - %v - %v - %v: need to check if this is versioned correctly.", groupVersion, resourceType, field, value)
+		// glog.Warningf("Field selector: %v - %v - %v - %v: need to check if this is versioned correctly.", groupVersion, resourceType, field, value)
```

Comment:
Commented rather than deleted so that if the entire file gets removed/deleted/renamed in a refactor, we'll get smacked with a conflict.

The previous carry for this deleted the lines, so a rebase silently dropped the commit when the rest of the file got deleted and recreated in a different package.